src/shortcuts.js

```diff
@@ -513,6 +513,17 @@ export const globalShortcuts = perma(() => [
     description: 'Navigate to Home.',
     keyboard: { key: 'h', shift: true, meta: true },
     exec: home
+  },
+
+  {
+    id: 'openShortcutPopup',
+    name: 'Open Shortcut Popup',
+    description: `Open the help screen which contains the tutorials and a list of all ${ isMobile ? 'gestures' : 'keyboard shortcuts'}.`,
+    keyboard: { key: '/', meta: true},
+    exec: e => {
+      window.scrollTo(0, 0)
+      store.dispatch({ type: 'showHelper', id: 'shortcuts' })
+    }
```

Comment:
Here's another commit from a different PR. Do you think you may have branched from `tutorial` instead `master`?

Reply:
Oh looks like I did. Good catch.

Reply:
You may have to put your new `git cherry-pick` skills to use!